.github/workflows/phpunit-mysql.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['8.1']
```

Comment:
lets also test against '8.2' or even '8.3' due to base image of php-fpm container by ionos